examples/cpp/Makefile

```diff
@@ -1,17 +1,23 @@
 MOLE = ../../src/cpp
 
-CXX ?= g++
+#ARMA = /Users/aneeshsrinivas/Documents/GitHub/armadillo-14.0.3
```

Comment:
Remove all of these extra "macOS" comments and commented code. It is confusing, as some macOS users might have libraries there.